### ISSUE TITLE
RP Community Slack Feedback: Identify where schemas are stored and enforced

### ISSUE THREAD

**Issue by Feediver1** [redpanda-data/documentation#677]
### Describe the Issue

See: https://redpandacommunity.slack.com/archives/C01B3QBLUTT/p1663927763781279

tl;dr
Mark
 [Sep 23rd at 5:09 AM](https://redpandacommunity.slack.com/archives/C01B3QBLUTT/p1663927763781279)
Hi, I have been looking at the docs about RedPanda and I see that it does not need a separate binary for schema management but I do not see any details on how it works or where schemas are stored / enforced?
5 replies

Mark
  [5 days ago](https://redpandacommunity.slack.com/archives/C01B3QBLUTT/p1663928107080289?thread_ts=1663927763.781279&cid=C01B3QBLUTT)
its ok, google helped me find the blog post on it. Can i suggest it gets linked or reference to the comparison page where it say you do schema's to make it easier to find

Mark
  [5 days ago](https://redpandacommunity.slack.com/archives/C01B3QBLUTT/p1663928855151149?thread_ts=1663927763.781279&cid=C01B3QBLUTT)
I see (from downloading the latest docker image) that you are now supporting AVRO and PROTOBUF schemas is there a timeline for JSON ?

Christian Nikolov
  [5 days ago](https://redpandacommunity.slack.com/archives/C01B3QBLUTT/p1663938350113689?thread_ts=1663927763.781279&cid=C01B3QBLUTT)
The schema registry its in the same execution just running on different port, at the moment I am looking for info with examples how to use it.

alex (redpanda)
:rp-favicon:  [5 days ago](https://redpandacommunity.slack.com/archives/C01B3QBLUTT/p1663938503355949?thread_ts=1663927763.781279&cid=C01B3QBLUTT)
Sure thing. 
[@Joyce Fee](https://redpandacommunity.slack.com/team/U039T4R4PJ7)
 and 
[@Michele Cyran](https://redpandacommunity.slack.com/team/U03JMAYQF2P)
 ^

alex (redpanda)
:rp-favicon:  [5 days ago](https://redpandacommunity.slack.com/archives/C01B3QBLUTT/p1663938525840419?thread_ts=1663927763.781279&cid=C01B3QBLUTT)
There are blog posts on it

DOCS: Ensure that the requested information and links are correct/exist in our docs. Thx.

**Comment by shorgi** [comment]
Closing as duplicate of #842
